docs/server-setup.md

```diff
@@ -66,10 +66,11 @@ To run your tests against a remote Selenium Server, you will need an account wit
 In your config file, set these options:
  - `sauceUser` -  The username for your Sauce Labs account.
  - `sauceKey` -  The key for your Sauce Labs account.
-
+ - `sauceSeleniumAddress` - default: ondemand.saucelabs.com:80/wd/hub ; travis: localhost:4445/wd/hub
```

Comment:
Since most users won't need to change this, I hesitate to mention it here - and if so, we should use a full sentence to describe when you would want to change it and what 'travis' means in this context. If you'd like to do this, would you mind making it a new pull request (just so that things are separated)? Thanks!